src/backend/utils/misc/guc_gp.c

```diff
@@ -5739,8 +5740,8 @@ check_gp_workfile_compression(bool *newval, void **extra, GucSource source)
 	return true;
 }
 
-void
-DispatchSyncPGVariable(struct config_generic * gconfig)
+static void
+dispatch_sync_pg_variable(struct config_generic * gconfig, bool is_sync)
```

Comment:
why not DispatchSyncPGVariableImpl or DispatchSyncPGVariableInternal?

Reply:
What do you think about DispatchSyncPGVariableForSync() function name? It stands in line with other "...ForSync()" functions that were added, but sounds silly